tests/unit/helpers/date.spec.js

```diff
@@ -63,4 +64,12 @@ describe('Date helpers', () => {
   it('getBCTimestamp', () => {
     expect(typeof getBCTimestamp() === 'string').toBe(true);
   });
+
+  it('getAge', () => {
+    let input = new Date();
+    input.setFullYear( input.getFullYear() - 10 );
+    expect(getAge(input)).toBe(10);
+    input.setDate( input.getDate() + 1 );
+    expect(getAge(input)).toBe(9);
```

Comment:
How does this function handle `undefined`, `null`, and `NaN` input parameters? I think you might need to add an `if` statement to the function which checks if `!(date instanceof Date)`, then return early the input parameter before you do your calculation logic.

Reply:
changed to return undefined if given improper input